nilearn/input_data/nifti_spheres_masker.py

```diff
@@ -0,0 +1,234 @@
+"""
+Transformer for computing seeds signals.
+"""
+
+import numpy as np
+
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.externals.joblib import Memory
+
+from .. import _utils
+from .._utils import logger, CacheMixin
+from .._utils.niimg_conversions import check_niimg
+from .. import signal
+from .. import image
+from .. import masking
+
+
+def _signals_from_seeds(seeds, niimg, radius=None, mask_img=None):
+    """ Note: this function is sub-optimal for small radius
+    """
+    n_seeds = len(seeds)
+    niimg = check_niimg(niimg)
+    shape = niimg.get_data().shape
+    affine = niimg.get_affine()
+    if mask_img is not None:
+        mask_img = check_niimg(mask_img, ensure_3d=True)
+        mask_img = image.resample_img(mask_img, target_affine=affine,
+                                      target_shape=shape[:3],
+                                      interpolation='nearest')
+        mask, _ = masking._load_mask_img(mask_img)
+    signals = np.empty((shape[3], n_seeds))
+    # Create an array of shape (3, array.shape) containing the i, j, k indices
+    # in voxel space
```

Comment:
The comment `Create an array of shape (3, array.shape)` is above the definition of `coords`.  `Coords` actually has a 4th row of all ones, so the comment isn't strictly true.  I thought it might be worth mentioning that 4th (constant) row in the comment.